doc/extending/xlsx/xlsx_import.md

```diff
@@ -61,7 +61,7 @@ NW = no warning generated for these conversions.
   - diagonal borders
 - Align :
   - some horizontal alignments. We only support left/right/center.
-  - vertical alignement
+  - some vertical alignements. We only support top/center/bottom.
```

Comment:
We should mention that the unsupported alignments will be converted into supported ones. Same for horizontal alignment.